config/routes.rb

```diff
@@ -111,27 +113,41 @@
 
       get "review-ect-details", action: :new
       post "review-ect-details", action: :create
+      get "change-review-ect-details", action: :new
+      post "change-review-ect-details", action: :create
 
       get "email-address", action: :new
       post "email-address", action: :create
+      get "change-email-address", action: :new
+      post "change-email-address", action: :create
 
       get "start-date", action: :new
       post "start-date", action: :create
+      get "change-start-date", action: :new
+      post "change-start-date", action: :create
 
       get "state-school-appropriate-body", action: :new
       post "state-school-appropriate-body", action: :create
+      get "change-state-school-appropriate-body", action: :new
+      post "change-state-school-appropriate-body", action: :create
 
       get "independent-school-appropriate-body", action: :new
       post "independent-school-appropriate-body", action: :create
+      get "change-independent-school-appropriate-body", action: :new
+      post "change-independent-school-appropriate-body", action: :create
 
       get "programme-type", action: :new
       post "programme-type", action: :create
+      get "change-programme-type", action: :new
```

Comment:
My PR add PT and LP change links, so there is a duplicate here and the LP ones could be reordered to follow your pattern.

Reply:
Good catch! I didn't encounter any conflicts in the routes file and didn't notice the duplicates when I rebased my branch. I pushed another commit to remove them.